Task-1-Data-Extraction-Preprocess/books/যা_দেখেছি_যা_বুঝেছি_যা_করেছি/যা_দেখেছি_যা_বুঝেছি_যা_করেছি_1_clean.txt

```diff
@@ -0,0 +1,2157 @@
+দেখতে পেয়েছিলাম হঠাৎ করেই যেন তা মিলিয়ে গেল। উৎকষ্ঠায় অনিশ্চিত
+ভবিষ্যতের দিকে চেয়ে থাকলাম। পাকিস্তানের দুই অংশের দুই প্রভাবশালী নেতার
+অনুপস্থিতিতে ২৬শে ফেব্রুয়ারী ১৯৬৯ রাওয়ালপিন্ডিতে গোলটেবিল বৈঠক বসে।
+প্রথম বৈঠকের পর অধিবেশন যুলতবী হয়ে যায় ১০ই মার্চ পর্যন্ত। ১০ই মার্চের
+অধিবেশনে শেখ মুজিব তার ওয়াদার বরখেলাপ করলেন। ছাত্র সংগ্রাম পরিষদের ১১
+দফার পরিবর্তে শেখ মুজিব তার ৬ দফার দাবি উতাপন করলেন। কথা ছিল ১১
+দফাই হবে বৈঠকে ডাক এর আলোচনার ভিত্তি। কোন বিশেষ পার্টি প্রোগ্রাম বা দাবি
+সেখানে উত্থাপিত হবে.না। কিন্তু মুজিব সমঝোতার বরখেলাপ করে ৬ দফার দাবি
+উত্থাপন করায় নেতাদের মধ্যে অনৈক্যের সৃষ্টি হল। ডাক ভেঙ্গে গেল। বৈঠকে কোন
+রাজনৈতিক সিদ্ধান্ত-গ্রহণ করা সম্ভব হল না। ১৩ই মার্চ প্রেসিডেন্ট আইয়ূব সভাপতির
+ভাষণে দুইটি প্রস্তাব মেনে নেন -
+
+১) প্রাপ্ত বয়স্কদের ভোটাধিকার ভিত্তিক প্রত্যক্ষ নির্বচেন এবং
+২) পার্লামেন্টারী শাসন পুনঃপ্রবর্তন।
+
+যা দেখেছি, যা বুঝেছি, যা করেছি ১৫
+
+ 
+
+১৪ই মার্চ শেখ মুজিব ও তার ৯ সদস্য ঢাকায় ফিরে আসেন ৷ ততক্ষণে নতুন
+ষড়যন্ত্রের সূচনা হয়েছে।
+
+সারা দেশব্যাপী আইয়ূব বিরোধী বিক্ষোভ তখন তুঙ্গে । আইয়ূবের ব্যক্তিগত ভাবমূর্তি
+ও জনপ্রিয়তা ক্রমশঃ কমে আসছে। ১৯৬৮ সালের জানুয়ারীতে প্লুমনারী
+এমবোলিজমে আক্রান্ত হওয়ার পর তিনি শারীরিকভাবে দুর্বল হয়ে পড়েন । এতে করে
+তার কর্মক্ষমতা বিশেষভাবে .ক্বাস পায়। তিনি আর আগের মত সবখানে যেতে
+পারছেন না৷ গণসংযোগ রক্ষা করতে পারছেন না প্রয়োজন মত । সামরিক বাহিনীর
+সদস্যবৃন্দ ও ইউনিটগুলো থেকে শারীরিক অসুস্থতার কারণে তিনি বিচ্ছিন্ন হয়ে
+পড়ছিলেন দিনকে দিন। শাসনকার্য পরিচালনার জন্য সম্পূর্ণভাবে নির্ভরশীল হয়ে
+পড়েছিলেন গুটিকতক জ্বী-হুজুর টাইপ আমলার উপর । প্রতিদিনের রুটিন কাজগুলো
+সংক্ষিপ্ত করে সম্পাদন করতে হচ্ছিল তাকে ।
+
+এ অবস্থার পূর্ণ সুযোগ গ্রহণ করলেন চক্রান্তকারী ক্ষমতা লিদ্গু জেনারেলগণ ৷ তাদের
+সাথে হাত মিলান জনাব ভুট্টো এবং আইয়ূব বিরোধী বেসামরিক আমলাতন্ত্রের
+সদস্যবৃন্দ। আইয়ূব বিরোধী এই চক্রান্তের নেতৃত্ব দেন তারই নিযুক্ত সেনা প্রধান
+জেনারেল ইয়াহিয়া খান। ১৯৫৮ সালে জেনারেল আইয়ূৰ খান যখন সেনা প্রধান
+হিসাবে জনাব ইস্কান্দর মির্জাকে সরিয়ে রাষ্ট্র ক্ষমতা দখলের চক্রান্ত আটছিলেন তখন
+তরুণ সেনা অফিসার ইয়াহিয়া খান ছিলেন তার বিশেষ অনুগত একজন ষ্টাফ
+অফিসার । অত্যন্ত দক্ষতার সাথে আইয়ূবের ক্ষমতা দখলের নীল নকশা প্রণয়ন করে
+রাষ্ট্রপতির বিশেষ আস্থাভাজন হয়ে উঠেন তিনি। তারই ফলশ্রুতিতে ১৯৬৬ সালে
+জেনারেল মুসা খান সামরিক বাহিনী প্রধানের পদ থেকে অব্যাহতি নেবার পর
+অনেককে সুপারসিড করে প্রেসিডেন্ট ইয়াহিয়া খানকে সেনা প্রধান হিসাবে নিয়োগ
+করা হয়। আইয়ূব খানের ধারণায় ইয়াহিয়া ছিলেন সহজ হালকা মেজাজের আরামপ্রিয়
+একজন সামরিক অফিসার ৷ যার কোন রাজনৈতিক উচ্চাভিলাস নেই৷ ভেবেছিলেন,
+স্বীয় শাসন টিকিয়ে রাখতে জেনারেল ইয়াহিয়ার মাধ্যমে সেনা বাহিনীর উপর তিনি
+তার নিয়ন্ত্রণ রাখতে পারবেন অতি সহজে ৷ কিন্তু তার সে ভাবনা মিথ্যে প্রমাণিত হয়।
+সার্বিক পরিস্থিতির সুযোগ নিয়ে একদিকে জেনারেলদের নিয়ে ইয়াহিয়া খান তার
+ক্ষমতা দখলের নীল নকশা নতুন করে তৈরি করতে গোপন বৈঠকে মিলিত হতে
+থাকেন; অপরদিকে জেনারেল পীরজাদার মাধ্যমে ভুষ্টোকে উৎসাহিত করেন আইয়ূব
+বিরোধী গণআন্দোলন জোরদার করে তোলার জন্য । জেনারেলদের পরোক্ষ
+আশীরাদপুষ্ট হয়েই জনাব ভুষ্টো সরাসরিভাবে রাজনৈতিক অঙ্গনে আত্মপ্রকাশ করেন
```

Comment:
আশীরাদপুষ্ট should be আশীর্বাদপুষ্ট. in avro it will be 'ashirrbadopuShTo'

Reply:
fixed